examples/chip-tool/commands/clusters/Commands.h

```diff
@@ -25,8 +25,24 @@
 
 #include <support/SafeInt.h>
 
-bool ReadAnyType(uint8_t *& message)
-{
+#define CHECK_MESSAGE_LENGTH(value)                                                                                                \
+    if (!chip::CanCastTo<uint16_t>(value))                                                                                         \
+    {                                                                                                                              \
+        ChipLogError(chipTool, "CHECK_MESSAGE_LENGTH expects a uint16_t value, got: %d", value);                                   \
+        return false;                                                                                                              \
+    }                                                                                                                              \
+                                                                                                                                   \
+    if (messageLen < value)                                                                                                        \
+    {                                                                                                                              \
+        ChipLogError(chipTool, "Unexpected response length: %d", messageLen);                                                      \
+        return false;                                                                                                              \
+    }                                                                                                                              \
+                                                                                                                                   \
+    messageLen = static_cast<uint16_t>(messageLen - static_cast<uint16_t>(value));
```

Comment:
I guess this line worth a comment. I add to do that to make GCC happy with `-WConversion`